docs/devguide/debugging_guide.rst

```diff
@@ -36,44 +36,70 @@ Default discovery workflow
     title Default Discovery Workflow
     Server->RackHD: DHCP from PXE(nic or BIOS)
     RackHD->Server: ISC DHCP response with IP
+    note over RackHD:
+        If the node is already "known",
+        it will only respond if there's an active workflow
+        that's been invoked related to the node
+    end note
     RackHD->Server: DHCP-proxy response with bootfile
-    note right of RackHD: If the node is already "known", it will only respond if there's an active workflow that's been invoked related to the node
     Server->RackHD: Request to download bootfile via TFTP
     RackHD->Server: TFTP sends requested file (monorail.ipxe)
-    note left of Server: Server loads monorail.ipxe and initiates on bootloader
+    note over Server:
+        Server loads monorail.ipxe
+        and initiates on bootloader
+    end note
     Server->RackHD: IPXE script requests what to do from RackHD (http)
-    note right of RackHD:
-        multiline
+    note over RackHD:
         RackHD looks up IP address of HTTP request from iPXE script to find the node via its mac-address.
-        If the node is already "known", it will only respond if there's an active workflow that's been invoked related to the node
-        If the node isn't known, it will create a workflow (default is the workflow 'Graph.Sku.Discovery') and respond with an iPXE script to initiate that
-        end note
+        1) If the node is already "known", it will only respond if there's an active workflow
+        that's been invoked related to the node.
+        2) If the node isn't known, it will create a workflow (default is the workflow 'Graph.Sku.Discovery')
```

Comment:
Known is confusing, suggest 'discovered'

Reply:
How about ""Default workflow is 'Graph.Sku.Discovery'"?

Reply:
See the above.